lib/DxilValidation/DxilValidation.cpp

```diff
@@ -3938,7 +3977,7 @@ static void ValidateShaderFlags(ValidationContext &ValCtx) {
   unsigned valMajor, valMinor;
   ValCtx.DxilMod.GetValidatorVersion(valMajor, valMinor);
   if (DXIL::CompareVersions(valMajor, valMinor, 1, 5) >= 0 &&
-      DXIL::CompareVersions(valMajor, valMinor, 1, 8) < 0 &&
+      DXIL::CompareVersions(valMajor, valMinor, 1, 8) != 0 &&
```

Comment:
I don't think this change is correct.  This is a WAR for a bug fixed in DXIL 1.8.